artifact-upload/action.yml

```diff
@@ -26,4 +26,4 @@ runs:
     - name: ci/artifact-upload
       shell: bash
       run: |
-        aws s3 sync ${{ inputs.artifacts }} s3://${{ inputs.bucket }}
+        aws s3 cp ${{ inputs.artifacts }} s3://${{ inputs.bucket }} --acl public-read --cache-control no-cache
```

Comment:
Why do we switch from `aws s3 sync` to `aws s3 cp` ?

Reply:
As I am mentioning I just replicate the CircleCI but that's the reason. Sync is definitely more efficient but I don't want to disrupt for now any use case the team has. We will revisit with the new year everything around CI.
```
aws s3 cp will copy all files, even if they already exist in the destination area. It also will not delete files from your destination if they are deleted from the source.

aws s3 sync looks at the destination before copying files over and only copies over files that are new and updated.
```